thamos/config.py

```diff
@@ -298,6 +300,31 @@ def list_runtime_environments(self):
         """List available runtime environments."""
         return self.content.get("runtime_environments", [])
 
+    def set_runtime_environment(
```

Comment:
Is this going to sync only the Thoth config updated on disk? Because after creating overlays/{kernel_name} with Pipfile and Pipfile.lock I want to update the thoth.yaml on disk to have the new runtime environment used in thoth resolution engine. WDYT?

Reply:
OK, sounds reasonable. We can add a method which writes the config file to disc once all the operations are done. :+1: